sdk/include/psdk/winbase.h

```diff
@@ -3931,6 +3931,34 @@ CopyFile2(
 
 #endif /* _WIN32_WINNT >= 0x0601 */
 
+#if _WIN32_WINNT >= 0x0600
+
+WINBASEAPI
+VOID
+WINAPI
+InitOnceInitialize(
+    _Out_ PINIT_ONCE InitOnce
+    );
+
+WINBASEAPI
+BOOL
+WINAPI
+InitOnceBeginInitialize(
+    _Inout_ LPINIT_ONCE lpInitOnce,
+    _In_ DWORD dwFlags,
+    _Out_ PBOOL fPending,
+    _Outptr_opt_result_maybenull_ LPVOID *lpContext);
+
+WINBASEAPI
+BOOL
+WINAPI
+InitOnceComplete(
+    _Inout_ LPINIT_ONCE lpInitOnce,
+    _In_ DWORD dwFlags,
+    _In_opt_ LPVOID lpContext);
+
+#endif /* _WIN32_WINNT >= 0x0600 */
```

Comment:
Why isn't the InitOnceExecuteOnce function below inside this _WIN32_WINNT guard?

Reply:
Because `InitOnceExecuteOnce` is currently used by some components like urlmon.dll

Reply:
Shouldn't then those components be compiled with the _WIN32_WINNT defined to 6+? (ok this is out of scope of this PR)

Reply:
Nope, ole32.dll, urlmon.dll, winhttp.dll, wininet.dll invoked `InitOnceExecuteOnce`, I once moved `InitOnceExecuteOnce` inside `_WIN32_WINNT >= 0x0600` and get compile-time errors.
So I guess this is why we have `InitOnceExecuteOnce` only before this PR, and didn't guarded by `_WIN32_WINNT >= 0x0600`